server.go

```diff
@@ -13,7 +13,7 @@ import (
 	"time"
 )
 
-// Maximum number of TCP queries before we close the socket.
+// Default maximum number of TCP queries before we close the socket.
 const maxTCPQueries = 128
```

Comment:
This should probably be renamed `defaultMaxTCPQueries`, but I can't say I love the name. Maybe it's fine as it is :man_shrugging:.